_posts/2017/2017-02-20-creating-a-sales-analytics-app-with-ruby-on-rails.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title: "Creating a Sales Analytics App with Ruby on Rails"
+date: "2017-02-20 07:00:00"
+image: blog-header/hamburg-port-rails.jpg
```

Comment:
I liked the fact that it were Hamburg Rails. But if you say it's better to change, I trust you, so just choose the image that you think it fits the best